glusterd2/snapshot/store-utils.go

```diff
@@ -92,27 +94,47 @@ func AddOrUpdateSnap(snapInfo *Snapinfo) error {
 // Snapinfo object
 func GetSnapshot(name string) (*Snapinfo, error) {
 	var snap Snapinfo
-	resp, e := gdstore.Store.Get(context.TODO(), snapPrefix+name)
+	resp, e := gdstore.Get(context.TODO(), snapPrefix+name)
 	if e != nil {
 		log.WithError(e).Error("Couldn't retrive volume from store")
 		return nil, e
 	}
 
 	if resp.Count != 1 {
-		log.WithField("volume", name).Error("volume not found")
-		return nil, errors.New("volume not found")
+		log.WithField("snapshot", name).Error("snapshot not found")
+		return nil, errors.New("Snapshot not found")
 	}
 
 	if e = json.Unmarshal(resp.Kvs[0].Value, &snap); e != nil {
-		log.WithError(e).Error("Failed to unmarshal the data into volinfo object")
+		log.WithError(e).Error("Failed to unmarshal the data into snapinfo object")
 		return nil, e
 	}
 	return &snap, nil
 }
 
 //DeleteSnapshot passes the snap path to store to delete the snap object
 func DeleteSnapshot(snapInfo *Snapinfo) error {
-	_, e := gdstore.Store.Delete(context.TODO(), GetStorePath(snapInfo))
+	var vol *volume.Volinfo
+	_, e := gdstore.Delete(context.TODO(), GetStorePath(snapInfo))
+	if e != nil {
+		return e
+	}
+
+	vol, e = volume.GetVolume(snapInfo.ParentVolume)
+	if e != nil {
+		return e
+	}
+
+	//TODO change this when label based snapshots are in.
+	e = errors.New("Snap is not found in the volinfo")
```

Comment:
error message should start with smaller case

Reply:
done